src/Analysis/Ast/Impl/Types/PythonClassType.Generics.cs

```diff
@@ -23,13 +23,14 @@
 
 namespace Microsoft.Python.Analysis.Types {
     internal partial class PythonClassType {
```

Comment:
Add IGenericType here, since the interface is implemented below?

Reply:
`IPythonClassType : IPythonClassMember, IGenericType`

Reply:
Aha, yes.

Reply:
`IPythonClassType` is derived from it